rulesets/washington-post-securedrop-ruleset.xml

```diff
@@ -0,0 +1,5 @@
+<ruleset name="The Washington Post">
+	<target host="washingtonpost.securedrop.tor.onion" />
+	<rule from="^http[s]?://washingtonpost.securedrop.tor.onion"
+    to="http://vfnmxpa6fo4jdpyq3yneqhglluweax2uclvxkytfpmpkp5rsl75ir5qd.onion" />
```

Comment:
Like NYT, WaPo redirects to HTTPS, but because we do not record this fact in the directory, our script always generates `http://` URLs:
https://github.com/freedomofpress/securedrop-https-everywhere-ruleset/blob/1b5122653911135446a2df598310bee37b20669e/sddir.py#L10

If we want to change this behavior I think it should be done at the ruleset generation level; I've filed https://github.com/freedomofpress/securedrop.org/issues/832 to discuss. Since we also point to `http://` for NYT already, I don't consider this a reason to put this PR on hold.